powershell/SecretSanta.ps1

```diff
@@ -0,0 +1,43 @@
+﻿# SecretSanta.ps1
+# Script that accepts list of names and then determines a secret
+# santa match for each person
+
+Param(
+    [Parameter(Mandatory=$true)][string[]]$names
+)
+
+# Create function that returns $true if for every value in $listA,
+# the corresponding value at the same position in $listB is not the 
+# same. Returns $false otherwise
+Function Compare-Lists ( [string[]]$listA, [string[]]$listB )
```

Comment:
would be better if called "check for loops" or something similar. because it looks for the loops.